.travis/test_script.sh

```diff
@@ -0,0 +1,15 @@
+
+echo 'setup puppetlabs apt repo'
+wget https://apt.puppetlabs.com/puppet5-release-trusty.deb
+sudo dpkg -i puppet5-release-trusty.deb
+sudo apt-get update
+
+echo 'install pdk'
+sudo apt-get install pdk
+
+PR_DIR=$PWD
+
+pdk new module pr_new_modules --template-url=$PR_DIR --skip-interview
+pdk new module normal_module --skip-interview
+cd normal_module
+pdk convert --template-url=$PR_DIR --skip-interview --force
```

Comment:
Not sure what the expected behavior of `--skip-interview` on `pdk convert` is but it raised a question so I had to add `--force`.  

It makes me wonder if `--skip-interview` even does anything useful on `pdk convert`.